### DISCUSSION TITLE
Compatibility with rainmeter + windhawk.

### DISCUSSION THREAD

**Discussion by Vortigern-The-Grey** [eythaann/Seelen-UI#75] [Q&A] [answered]
Does Seelen UI work with Windhawk customizations and Rainmeter widgets?

**Comment by eythaann** [comment, marked as answer]
***Windhawk***: Windhawk is the opposite of the philosophy we follow in Seelen. Seelen aims not to modify the system or system files at all, whereas that’s precisely what Windhawk is for. Therefore, we won’t be implementing any widgets or features related to Windhawk.

***Rainmeter:*** You can combine Seelen with any other customization apps, so yes, you can use Seelen UI along with Rainmeter. However, we won’t offer any features related to Rainmeter since Seelen UI will have its own widget system in version v2.

**Reply by Vortigern-The-Grey** [reply]
Thank you!
